cetmix_tower_server_notify_backend/readme/CONFIGURE.md

```
@@ -1 +1 @@
-No configuration is required.
+Please refer to the [official documentation](https://cetmix.com/tower) for detailed configuration instructions.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)